deploy/README.md

```diff
@@ -23,15 +32,6 @@ Set an environment variable named `NGC_CLI_API_KEY` to your API Key.
 export NGC_CLI_API_KEY=$(cat ~/API_KEY)
 ```
 
-### Login to nvcr.io
```

Comment:
This should stay in the README.  The docker compose will fail to pull containers if not logged in to nvcr.io.